src/client/utils/validations.js

```diff
@@ -180,14 +212,18 @@ function isValidVeteranAddress(data) {
       isValidField(isValidPhone, data.mobilePhone);
 }
 
+function isValidFinancialDisclosure(data) {
+  return isNotBlank(data.provideFinancialInfo.value);
+}
+
 function isValidSpouseInformation(data) {
```

Comment:
Yes, definitely, these validations should be updated. I was waiting to do anything with the spouse section because the conditional logic isn't in place yet. I'll add a TODO for now here.